004-governance.Rmd

```diff
@@ -0,0 +1,153 @@
+---
+output: github_document
+---
+
+# Tidyup 4: governance model
+
+**Champion**: Hadley Wickham  
+**Co-champion**: Tracy Teal  
+**Status**: Proposa
+l
```

Comment:
```suggestion
**Status**: Proposal
```

Reply:
This is a bug with the visual editor that I'm still trying to reprex for JJ 😞